maestro/start-helm-cluster.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
this file shouldn't be here, otherwise, it would be included in every chart

Reply:
these are not included in the chart directory, the helm chart is /maestro/chart 
these are outside so shouldn't be included